repobuilder/job_test.go

```diff
@@ -73,3 +73,60 @@ func TestProcessPackages(t *testing.T) {
 	assert.NoError(t, j.processPackages(ctx))
 	assert.Len(t, j.PackagePaths, 6)
 }
+
+func TestGetPackageLocation(t *testing.T) {
+	for _, test := range []struct {
+		name             string
+		version          string
+		expectedLocation string
+	}{
+		{
+			name:             "LegacyReleaseCandidate",
+			version:          "4.2.5-rc1",
+			expectedLocation: "testing",
+		},
+		{
+			name:             "LegacyDevelopmentBuild",
+			version:          "4.2.5-pre-",
+			expectedLocation: "development",
+		},
+		{
+			name:             "LegacyDevelopmentSeries",
+			version:          "4.2.5",
+			expectedLocation: "4.2",
+		},
+		{
+			name:             "LegacyProductionSeries",
```

Comment:
This is the development series, not production series.

Reply:
The minor is odd, which means it is a  production series: 

Current scheme: major.minor.patch[-rcN]

- Development series: minor is even

- Production series (i.e. non-development series): minor is odd

- -rcN: release candidates

- Commits in the production series are always released. Commits in the development series can be released (but usually aren’t).